product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx

```diff
@@ -20,7 +20,7 @@ You can analyze the Wait States data on multiple levels by narrowing down your s
 
 Prerequisite:
 
--   For PostgreSQL, you need to install `edb_wait_states_<X>` package from `edb.repo` where `<X>` is the version of PostgreSQL Server. You can refer to [EDB Build Repository](https://repos.enterprisedb.com/) for the steps to install this package. For Advanced Server, you need to install `edb-as<X>-server-edb-modules`, Where `<X>` is the version of Advanced Server.
+-   For PostgreSQL, you need to install `edb_wait_states_<X>` package from `edb.repo` where `<X>` is the version of PostgreSQL Server. You can refer to [EDB Build Repository](https://repos.enterprisedb.com/) for the steps to install this package. For Advanced Server, you must install EDB wait states; see [EDB Wait States Background Worker (EWSBW)](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states-background-worker-ewsbw) under [EDB wait states](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states).
```

Comment:
What is `edb.repo`? some kind of dynamic variable? It was there before so I assume it's fine.